packages/preferences/src/browser/preference-service.spec.ts

```diff
@@ -239,4 +258,42 @@ describe('Preference Service', function () {
             stub.restore();
         });
     });
+
+    it('Should store preference when settings file is empty', (done: any) => {
```

Comment:
fyi: you can use `async () => { ...` here to make it more readable.

Reply:
done

Reply:
Thanks!